src/twisted/test/test_twistd.py

```diff
@@ -994,7 +994,9 @@ def setgid(gid):
         self.patch(os, "setgid", setgid)
 
         options = twistd.ServerOptions()
-        options.parseOptions(["--nodaemon", "--uid", str(wantedUid)])
+        options.parseOptions(
+            ["--nodaemon", "--uid", str(wantedUid), "--pidfile={}".format(pidFile)]
```

Comment:
Let's be stylistically consistent about how we treat the arguments.
```suggestion
            ["--nodaemon", "--uid", str(wantedUid), "--pidfile", pidFile]
```